### PR TITLE
Add in cookie security

### PR DESCRIPTION
**Pull request by TawneeOwl**
## What does this pull request do?

Adds in security to CLA_Public to the best it can possibly be without upgrading. Unsafe-eval is being used due to the webpack requiring it to run and not being able to take a nonce.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- https://dsdmoj.atlassian.net/browse/LGA-2735

### CODE REVIEW

File: cla_public/app.py [hunk 1]
```diff
@@ -31,6 +32,85 @@
...
+            "data:",
+            "*.googletagmanager.com",
+            "*.analytics.google.com",
+            "*.google.co.uk",
+            "*.g.doubleclick.net",
+            "*.google-analytics.com",
+        ],
+        "object-src": "'self'",
+        "script-src": [
+            "'unsafe-eval'",
```
Comment (BenMillar-CGI) [inline cla_public/app.py:53]: Would it be possible to leave a comment here explaining why `unsafe-eval` is enabled and that it should be removed when it is no longer required?

Comment (Pheonnexx) [reply cla_public/app.py:53]: agreed!

**Review by digitalali-moj — DISMISSED** [review]
Thanks for this!

File: cla_public/app.py (hunk 1 above)
Comment (TawneeOwl) [reply cla_public/app.py:53]: Updated

**Review by psweeting1 — APPROVED** [review]
LGTM just make sure to SQAUSH on merge :D

File: cla_public/app.py [hunk 2]
```diff
@@ -31,6 +32,91 @@
...
     app = Flask(__name__)
+    # Adds security to Flask, unsafe eval is required to use the webpack
+    csp = {
+        "default-src": ["'self'", "*.googletagmanager.com"],
+        "img-src": [
+            "'self'",
+            "*.googleapis.com",
+            "*.gstatic.com",
+            "*.google.com",
+            "*.googleusercontent.com",
```
[the remark this answers is not in the document]
Comment (TawneeOwl) [reply cla_public/app.py:43]: This is required to use the google maps screen as per their CSP https://developers.google.com/maps/documentation/javascript/content-security-policy

**Review by BenMillar-CGI — APPROVED** [review]
LGTM 👍

**Pull request merged**